index.html

```diff
@@ -4,170 +4,223 @@
+    </div>
 
-        <footer>
-            <div class="item-navigation-wrapper">
-                <img src="images/logo footer.svg" class="logo-small">
-                <p class="navigation-paragraph paragraph-logo">
-                    Lorem ipsum dolor sit amet consectetur adipiscing elit sed do eiusmod tempor incididunt ut labore et dolore magna aliqua Ut enim ad minim veniam.
-                </p>
-                <div class="icons-large-screen">
-                    <a href="Http://facebook.com" target="_blank"><img src="images/fb.svg"></a>
-                    <a href="Http://instagram.com" target="_blank"><img src="images/insta footer.svg"></a>
-                    <a href="Http://twitter.com" target="_blank"><img src="images/twitter footer.svg"></a>
-                </div>
-                <div class="icons-small-screen">
-                    <a href="Http://facebook.com" target="_blank"><img src="images/fb.svg"></a>
-                    <a href="Http://linkedin.com" target="_blank"><img src="images/insta.svg"></a>
-                    <a href="Http://instagram.com" target="_blank"><img src="images/insta footer.svg"></a>
-                    <a href="Http://google.com" target="_blank"><img src="images/google.svg"></a>
+    <div class="container-fluid mb-5 mt-lg-5 text-center text-lg-left">
+        <div class="row">
+            <div class="col-lg-7 mt-lg-5">
+                <div class="row">
+                    <div class="col-sm-6 col-md-6 col-lg-7">
+                        <div class="mx-lg-5">
+                            <img src="images/logo footer.svg" class="img-fluid p-3 mt-4 p-md-0 pb-md-3">
+                            <p class="paragraph-logo">
+                                Lorem ipsum dolor sit amet consectetur adipiscing elit sed do eiusmod tempor incididunt ut labore et dolore magna aliqua Ut enim ad minim veniam.
+                            </p>
+                            <div class="d-none d-md-block my-4">
+                                <a href="Http://facebook.com" target="_blank" class="pr-3"><img src="images/fb.svg"></a>
+                                <a href="Http://instagram.com" target="_blank" class="pr-3"><img src="images/insta footer.svg"></a>
+                                <a href="Http://twitter.com" target="_blank"><img src="images/twitter footer.svg"></a>
+                            </div>
+                            <div class="d-xs-block d-md-none p-2 mb-4">
+                                <a href="Http://facebook.com" target="_blank" class="p-3"><img src="images/fb.svg"></a>
+                                <a href="Http://linkedin.com" target="_blank" class="p-3"><img src="images/insta.svg"></a>
+                                <a href="Http://instagram.com" target="_blank" class="p-3"><img src="images/insta footer.svg"></a>
+                                <a href="Http://google.com" target="_blank" class="p-3"><img src="images/google.svg"></a>
+                            </div>
```

Comment:
Why can't we write this in one block?
You can add the same md/lg/xl properties for margins and padding as for display.
Example:
`mb-0 mb-lg-3 p-0 p-md-0`

Apply the same to all other places (don't duplicate content)